accelerator/constants.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+# DeepSpeed Team
+
+#A list of constants used in the DeepSpeed feature dictionary
+
+OP_ASYNC_IO = "async_io"
+OP_CCL_COMM = "deepspeed_ccl_comm"
+OP_CPU_ADAGRAD = "cpu_adagrad"
+OP_CPU_ADAM = "cpu_adam"
+OP_CPU_LION = "cpu_lion"
+OP_EVOFORMER_ATTN = "evoformer_attn"
+OP_FP_QUANTIZER = "fp_quantizer"
+OP_FUSED_ADAM = "fused_adam"
+OP_FUSED_LAMB = "fused_lamb"
+OP_FUSED_LION = "fused_lion"
+OP_INFERENCE_CORE_OPS = "inference_core_ops"
+OP_CUTLASS_OPS = "cutlass_ops"
```

Comment:
A general name is needed to cover the non-cuda devices?

Reply:
Hi @rogerxfeng8 - is there a specific one you're referencing?  I believe we call all devices (cuda and non-cuda) accelerators.